src/wdio/services/ServeStaticService.js

```diff
@@ -2,19 +2,74 @@ import serveStatic from '../../../scripts/serve/serve-static';
 import SERVICE_DEFAULTS from '../../../config/wdio/services.default-config';
 import Logger from '../../../scripts/utils/logger';
 
+const WebpackDevServer = require('webpack-dev-server');
+const webpack = require('webpack');
+const ip = require('ip');
+
 const { serveStatic: SERVE_STATIC_DEFAULTS } = SERVICE_DEFAULTS;
 const context = '[Terra-Toolkit:serve-static-service]';
 
+const displayServer = (localAddress, networkAddress) => {
+  Logger.log('Server started listening at', { context });
+  Logger.log(`* Local:            ${Logger.emphasis(localAddress)}`);
+  Logger.log(`* On your network:  ${Logger.emphasis(networkAddress)}`);
+};
+
+// Create a webpack dev server instance.
+const wds = (options) => {
+  const {
+    port, index, locale,
+  } = options;
+  const host = '0.0.0.0';
+  let { config } = options;
+  // if config is a function, execute it with prod mode if applicable.
+  if (typeof config === 'function') {
+    const env = { ...locale && { defaultLocale: locale } };
```

Comment:
Should this be 
```
 const env = { ...locale && { aggregateOptions: { defaultLocale: locale } } };
```

Reply:
No, this is the locale that consumers need to apply their generated sites.
For example in dev site:
https://github.com/cerner/terra-dev-site/blob/master/config/webpack/terra-dev-site.webpack.config.js#L60